commands/generate.js

```diff
@@ -0,0 +1,111 @@
+'use babel';
+
+var fs = require('fs');
+
+var getProjectRoot = function getProjectRoot() {
+	'use strict';
```

Comment:
Why are you putting this in every function? It's enough to define it on the top of the file, but use strict is moot anyways as you use `use babel`.